cfme/fixtures/service_fixtures.py

```diff
@@ -87,5 +87,5 @@ def order_service(appliance, provider, provisioning, vm_name, dialog, catalog, r
         partial_check=True)
     provision_request.wait_for_request(method='ui')
     assert provision_request.is_succeeded()
-    return catalog_item, provision_request
+    return catalog_item
 # TODO - remove request finally
```

Comment:
Shouldn't this line be removed also?

Reply:
No , the request still has to be removed :)